src/main/java/org/folio/consortia/controller/SharingInstanceController.java

```diff
@@ -0,0 +1,24 @@
+package org.folio.consortia.controller;
+
+import static org.springframework.http.HttpStatus.CREATED;
+
+import java.util.UUID;
+
+import org.folio.consortia.domain.dto.SharingInstanceAction;
+import org.folio.consortia.rest.resource.SharingApi;
+import org.folio.consortia.service.SharingInstanceService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+
+import lombok.RequiredArgsConstructor;
+
+@Controller
+@RequiredArgsConstructor
+public class SharingInstanceController implements  SharingApi{
```

Comment:
Lets rename service, controller, dao to ShareInstanceController, ShareInstanceService, ShareInstanceDao